spec-tests/src/test/java/org/candlepin/spec/ContentResourceSpecTest.java

```diff
@@ -254,7 +257,20 @@ public void shouldListContentPagedAndSorted() {
                 }
```

Comment:
We can actually remove this test entirely since we have better tests that cover this behavior.

Reply:
Removed